src/styles/_colors.scss

```diff
@@ -1,12 +1,12 @@
-html {
-	--base-color-background: #282c34;
+:root {
+	--base-color-background: white;
 	--base-color-link: #61dafb;
 	--base-color: white;
 	--base-color-button: #929292;
 }
 
-html[data-theme="dark"] {
-	--base-color-background: white;
+:root[data-theme="dark"] {
+	--base-color-background: #282c34;
 	--base-color-link: #fe016c;
 	--base-color: #232323;
 	--base-color-button: #273239;
```

Comment:
I think we got a misunderstood when I say the variables must be in the `:root`  What I want to say we have to declare all colors in the `:root` and later use them on the `html` element.
Some thing like this:
```css
/* Only CSS Custom Properties */
:root {
  --blue: #0096ca;
  --darkBg: #222;
  --lightBg: #fff;
}

html {
 --base-color-background: var(--lightBg);
} 

html[data-theme="dark"] {
 --base-color-background: var(--darkBg);
} 
```
```scss
/* CSS Custom Properties and SASS */
$blue: #0096ca;
$darkBg: #222;
$lightBg: #fff;

html {
 --base-color-background: #{$lightBg};
} 

html[data-theme="dark"] {
 --base-color-background: #{$darkBg};
} 
```